docs/user-guide/data-structures/creating-arrays-and-datasets.ipynb

```diff
@@ -0,0 +1,527 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Creating Arrays and Datasets\n",
+    "\n",
+    "There are several ways to create data structures in Scipp.\n",
+    "`scipp.Variable` is particularly diverse.\n",
+    "\n",
+    "The examples on this page only show some representative creation functions and only the most commonly used arguments.\n",
+    "See the linked reference pages for complete lists of functions and arguments.\n",
+    "\n",
+    "## Variable\n",
+    "\n",
+    "[Variables](../../generated/classes/scipp.Variable.rst) can be created using any of the dedicated [creation functions](../../reference/creation-functions.rst#creation-functions).\n",
+    "These fall into several categories as described by the following subsections.\n",
+    "\n",
+    "### From Python Sequences or NumPy Arrays\n",
+    "\n",
+    "#### Arrays: N-D Variables\n",
+    "\n",
+    "Variables can be constructed from any Python object that can be used to create a NumPy array or NumPy arrays directly.\n",
+    "See [NumPy array creation](https://numpy.org/doc/stable/user/basics.creation.html) for details.\n",
+    "Given such an object, an array variable can be created using [scipp.array](../../generated/functions/scipp.array.rst) (not to be confused with [data arrays](../../generated/classes/scipp.DataArray.rst)!)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "jupyter": {
+     "outputs_hidden": false
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import scipp as sc\n",
+    "v1d = sc.array(dims=['x'], values=[1, 2, 3, 4])\n",
+    "v2d = sc.array(dims=['x', 'y'], values=[[1, 2], [3, 4]])\n",
+    "v3d = sc.array(dims=['x', 'y', 'z'], values=[[[1, 2], [3, 4]],\n",
+    "                                             [[5, 6], [7, 8]]])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Alternatively, passing a NumPy array:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "jupyter": {
+     "outputs_hidden": false
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "a = np.array([[1, 2], [3, 4]])\n",
+    "v = sc.array(dims=['x', 'y'], values=a)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that *both* the NumPy array and Python lists are copied into the Scipp variable which leads to some additional memory usage.\n",
+    "\n",
+    "The `dtype` and unit of the variable are deduced automatically in the above cases:"
```

Comment:
While *I* know that by unit deduction you mean `dimensionless` or `None`, but I think this is unclear to the reader. Maybe we should avoid calling it deduction *of the unit*... it is deducing *whether* there *is* a unit, not *hat* the unit is?